docs/configuration.md

```diff
@@ -636,6 +636,8 @@ Apart from these, the following properties are also available, and may be useful
   <td>Long.MAX_VALUE</td>
   <td>
     The max number of chunks allowed to being transferred at the same time on shuffle service.
+    Note that new coming connections will be closed when the max number is hit. Client should
```

Comment:
change to say: allowed to be transferred

new coming should be new incoming.

Perhaps we could clarify a bit because Spark has a built in retry mechanism.   Could we rephrase to something like:   The client will retry according to the shuffle retry configs (see spark.shuffle.io.maxRetries and spark.shuffle.io.retryWait), if those limits are reached the task will fail with fetch failure.